src/components/map/MapService.js

```diff
@@ -362,6 +362,7 @@ goog.require('ga_urlutils_service');
           transparent: 'true'
         };
         return new Cesium.UrlTemplateImageryProvider({
+          minimumRetrievingLevel: window.minimumRetrievingLevel,
```

Comment:
if it will production code, then yes.

but I think this here is for testing different settings. once we have a final decision it will be fixed value. Or we keep parameter.

Let's discuss it next wee.